uaclient/cli.py

```diff
@@ -1795,9 +1795,7 @@ def wrapper(*args, **kwargs):
             sys.exit(1)
         except exceptions.UrlError as exc:
             if "CERTIFICATE_VERIFY_FAILED" in str(exc):
-                tmpl = messages.SSL_VERIFICATION_ERROR_CA_CERTIFICATES
-                if util.is_installed("ca-certificates"):
```

Comment:
Maybe we only have that `util.is_installed` function because of this functionality. If that is true, we can also remove it in my opinion

Reply:
great catch - removed